RibonDAO/core-api
# feat: creates migration and CRUD of Reports

# Description

Creates the migration and CRUD of reports.

## Type of change



## How to test

Check the existence of the reports table in the database and make a request to the /reports endpoint
